utils/forest_shim/src/randomness.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fvm_shared::randomness::Randomness as Randomness_v2;
```

Comment:
In other places, we have this naming convention: TypeV2, TypeV3, camelcase without underscore in between. It would be clean to have that convention here as well.

Reply:
I guess we are using `_v2` or `_v3`:

https://github.com/ChainSafe/forest/blob/a3604075c5d75cabe7625d50c2c0567434226d19/utils/forest_shim/src/address.rs#L4

https://github.com/ChainSafe/forest/blob/a3604075c5d75cabe7625d50c2c0567434226d19/utils/forest_shim/src/bigint.rs#L6

https://github.com/ChainSafe/forest/blob/a3604075c5d75cabe7625d50c2c0567434226d19/utils/forest_shim/src/econ.rs#L5

Reply:
i see, that's fine then